src/test/java/uk/gov/pay/api/it/CreatePaymentITest.java

```diff
@@ -43,9 +47,31 @@
     private static final String CREATED_DATE = ISO_INSTANT_MILLISECOND_PRECISION.format(TIMESTAMP);
     private static final Address BILLING_ADDRESS = new Address("line1", "line2", "NR2 5 6EG", "city", "UK");
     private static final CardDetails CARD_DETAILS = new CardDetails("1234", "123456", "Mr. Payment", "12/19", BILLING_ADDRESS, CARD_BRAND_LABEL);
-    private static final String SUCCESS_PAYLOAD = paymentPayload(AMOUNT, RETURN_URL, DESCRIPTION, REFERENCE, EMAIL);
+    private static final String SUCCESS_PAYLOAD = paymentPayload(AMOUNT, RETURN_URL, DESCRIPTION, REFERENCE);
     private static final String GATEWAY_TRANSACTION_ID = "gateway-tx-123456";
 
+    @Test
+    public void createCardPaymentWithMetadata() {
+        publicAuthMock.mapBearerTokenToAccountId(API_KEY, GATEWAY_ACCOUNT_ID, CARD);
+
+        CreateChargeRequestParams createChargeRequestParams = aCreateChargeRequestParams()
+                .withAmount(100)
+                .withDescription(DESCRIPTION)
+                .withReference(REFERENCE)
+                .withReturnUrl(RETURN_URL)
+                .withMetadata(Map.of("foo", "foo you", "fuh", "fuh you"))
```

Comment:
I think we agreed to honour the JSON type in the response e.g. String, Boolean or Number so in the ITest I've written I've made sure to include all three types and assert they're returned as such in the response (e.g. making sure not everything comes back as a String). Might be useful here too? E.g.
```
ExternalMetadata externalMetadata = new ExternalMetadata(
                Map.of("key1", true, "key2", 123, "key3", "string1"));
```
and then... 
```
.body("results[0].metadata.key1", is(true))
.body("results[0].metadata.key2", is(123))
.body("results[0].metadata.key3", is("string1"));
```

Reply:
yes good shout

Reply:
resolved